pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -57,6 +58,11 @@ type MongoDB struct {
 func (m *MongoDB) UpdateSuccess() {
 	m.Status.MongoURI = m.MongoURI()
 	m.Status.Phase = Running
+	m.Status.Version = m.Spec.Version
```

Comment:
we should not add the version to the status, given that it is already on the spec.